### ISSUE TITLE
Label Smoothing Cross Entropy

### ISSUE THREAD

**Issue by zzw-zwzhang** [linkedin/Liger-Kernel#81]
### 🚀 The feature, motivation and pitch

Will it support this function?

### Alternatives

N/A

### Additional context

N/A

**Comment by ByronHsu** [comment]
Yeah we should! We might have to refactor/rewrite cross entropy kernel a bit. context: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

**Comment by ByronHsu** [comment]
the feature is now supported! Thanks to @Tcc0403
